plugin.xml

```diff
@@ -17,7 +17,8 @@
     <platform name="ios">
       <header-file src="src/ios/Intercom.h" />
       <source-file src="src/ios/libIntercom.a" framework="true" compiler-flags="-ObjC" />
```

Comment:
I think it will as it's essentially the same as what is there now as i pulled those files out of the framework so I think the only thing different is the folder structure.